sources/engine/Stride.Assets/Media/SoundAssetCompiler.cs

```diff
@@ -41,7 +41,7 @@ public DecodeSoundFileCommand(string url, SoundAsset parameters, IAssetFinder as
             protected override async Task<ResultStatus> DoCommandOverride(ICommandContext commandContext)
             {
                 // Get path to ffmpeg
-                var ffmpeg = ToolLocator.LocateTool("ffmpeg.exe")?.ToWindowsPath() ?? throw new AssetException("Failed to compile a sound asset, ffmpeg was not found.");
+                var ffmpeg = ToolLocator.LocateTool("ffmpeg")?.ToWindowsPath() ?? throw new AssetException("Failed to compile a sound asset, ffmpeg was not found.");
```

Comment:
If possible, we should get rid of the `.ToWindowsPath()`.

Maybe create a method to `ToOSPath()` or `ToSystemPath()`.

Reply:
I think ToOSPath name seems legit, but to clarify, I would prefer to make separate PR for this issue, because it will affect a lot more places 👀

Reply:
Alright, I created new ToOSPath method then 😅. This only replace 'opposite' separators e.g. os is / and FullPath is \